src/test/java/fi/otavanopisto/kuntaapi/test/CasemMocker.java

```diff
@@ -0,0 +1,53 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class CasemMocker extends AbstractMocker {
+  
+  private static final String URL_PREFIX = "/casem/api/opennc/v1/";
+  
+  public CasemMocker mockSubnodes(Integer rootNode, Integer...subNodes) {
+    List<String> pathSegments = new ArrayList<>();
+    
+    pathSegments.add(String.format("Nodes(%d)", rootNode));
+    
+    if (subNodes != null) {
+      for (Integer parent : subNodes) {
+        pathSegments.add(String.format("SubNodes(%d)", parent));
+      }
+    }
+    
+    pathSegments.add("SubNodes()");
+    
+    String path = StringUtils.join(pathSegments, "/");
+    
+    String file = String.format("subnodes/%d.json", subNodes == null || subNodes.length == 0 ? rootNode : subNodes[subNodes.length - 1]);
+    
+    mockGetString(String.format("%s/%s", URL_PREFIX, path), "application/json", readCasemFile(file));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "application/json" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)